scripts/push-version.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+WAIT_TIME=10
+GITHUB_REMOTE=https://github.com/evergreen-ci/spruce
+
+git push $GITHUB_REMOTE
+
+echo -n "Waiting ${WAIT_TIME}s for Evergreen to pick up the version"
+
+i=0
+while [ $i -lt $WAIT_TIME ]
+  do echo -n .; sleep 1; i=$((i+1))
+done
```

Comment:
You can also do some carriage return hacking if you want to animate the countdown :) 
```suggestion
i=$WAIT_TIME
while [ $i -gt 0 ]
  do echo -en "Waiting $i s for Evergreen to pick up the version\r"; sleep 1; i=$((i-1))
done
```


https://github.com/evergreen-ci/spruce/assets/4605522/ba2dafd9-6fc8-41fb-a0fb-d7f1491ef4bc